tests/test_register.py

```diff
@@ -50,6 +50,17 @@ def test_register_invalid_password_validation(verified_user_status_type, anonymo
     assert executed["errors"]
 
 
+async def test_channels_register(
+    verified_user_status_type, captcha, verified_channels_app_communicator
```

Comment:
You don't need a verified consumer here I think

Reply:
You're right, I replaced it with unverified_channels_app_communicator, as it doesn't require authentication when registering.